client/src/partials/reference_group/reference_group.js

```diff
@@ -0,0 +1,126 @@
+angular.module('bhima.controllers')
+.controller('reference_group', [
+  '$scope',
+  '$translate',
+  '$http',
+  'validate',
+  'messenger',
+  'connect',
+  'appstate',
+  function ($scope, $translate, $http, validate, messenger, connect, appstate) {
+    var dependencies = {},
+        session = $scope.session = {};
+
+    dependencies.reference_groups = {
+      query : {
+        identifier : 'id',
+        tables : {
+          'reference_group' : {
+            columns : ['id', 'is_report', 'reference_group', 'text', 'position', 'section_bilan_id']
+          },
+          'section_bilan' : {
+            columns : ['text::section_bilan_txt']
+          }          
+        },
+        join : [
+          'reference_group.section_bilan_id=section_bilan.id'
+        ],
+      }
+    };
+
+    dependencies.section_bilan = {
+      query : {
+        tables : {
+          'section_bilan' : {
+            columns : ['id', 'text']
+          }
+        }
+      }
+    };
+
+    dependencies.enterprise = {
+      query : {
+        tables : {
+          'enterprise' : {
+            columns : ['currency_id']
+          },
+          'currency' : {
+            columns : ['id', 'symbol']
+          }
+        },
+        join : [
+            'enterprise.currency_id=currency.id'
+          ]
+        }
+    };
+
+    function startup (models) {
+      angular.extend($scope, models);
+    }
+
+    appstate.register('enterprise', function (enterprise) {
+      $scope.enterprise = enterprise;
+      validate.process(dependencies)
+      .then(startup);
+    });
+
+    $scope.delete = function (reference_group) {
+      var result = confirm($translate.instant('REFERENCE_GROUP.CONFIRM'));
+      if (result) {  
+        connect.basicDelete('reference_group', reference_group.id, 'id')
```

Comment:
Do not use `connect.basicDelete()`, it is deprecated.  You should use `connect.delete()` instead.